src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -89,7 +101,9 @@ const ActivityView = View.extend({
       ActionStateUpdated: StateUpdatedTemplate,
     };
 
-    return Templates[this.model.get('type')];
+    if (!Object.keys(Templates).includes(type)) return hbs``;
```

Comment:
I think `!Templates[type]` works here.  Also `.includes` doesn't work in IE11 without a polyfill.  I think we have it, but I don't want to rely on it.